### DISCUSSION TITLE
Distribution group reply to manual activity

### DISCUSSION THREAD

**Discussion by castanerINX** [AdhocAdam/smletsexchangeconnector#276] [Q&A]
Ok not sure if this is a bug or not but I can't seem to find any documentation on it.

We have a user who created a few manual activities to a distribution group. One of the users replied to the manual activity email to complete it. 

When I look at the manual activity it shows the distribution group has complete the manual activity instead of the person who approved/completed the activity. 

So the process is when a person creates an MA the activity implementer(in this case the DL) get's an email link. The user clicks on the link which replies with the [Completed] in the body of the email. So all they have to do is click send. 

Now it looks like the group approved it instead of the individual.

Any help would be appreciated.

**Comment by AdhocAdam** [comment]
Hey again @castanerINX! I'd say you've certainly spelled this one out:
- A Manual Activity has an Activity Implementer that is a Distribution Group
- SCSM Notification engages and sends to the Distro Group
- An individual user in that group replies to the effect of [Complete]
- The Manual Activity is completed, but the Activity Implementer remains the Distro Group
- Unlike the out of box Review Activity, there isn't a concept of a Reviewer/Voted By relationship. In which case you don't know _which_ person in the group completed the Manual Activity

The logic in question is seen here:
https://github.com/AdhocAdam/smletsexchangeconnector/blob/0f3edf7aae77b9c7e29830d473123e5df39aefc6/smletsExchangeConnector.ps1#L2058-L2105

But a bug or an enhancement...either way both well within the realm of being part of the next release/first release of the year. I can think of two ways this could be improved. But of course am open to any other suggestions you (or onlookers of this thread) might have.
- Introducing your own custom logic to the Invoke-AfterUpdateMA function
- Improving the Notes that are written on the _default_ portion of the switch

**Comment by infiniterecursion** [comment]
Just stumbled across this and was wondering, does the distribution group have it's own email address and is that the address the user is responding from?
